### PR TITLE
WINDUP-2177: Enrich mouse over information on tech punch card

### PR DESCRIPTION
**Pull request by m-brophy**
tooltip boxes on technology punchcard report not resizing to show full width of text contents.
https://issues.jboss.org/browse/WINDUP-2177

### CODE REVIEW

**Comment by m-brophy** [comment]
This is ready to review now

File: reporting/impl/src/main/resources/reports/templates/techReport-punchCard.ftl [hunk 1]
```diff
@@ -60,13 +60,16 @@
...
+                <#assign sortedRowTags = iterableToList(reportModel.rowsHolderTag.designatedTags) />
+                <#assign sortedRowTags = sortedRowTags?sort_by("title") />
 
-                <#-- MatrixAndAggregated {
-                       countsOfTagsInApps,  // Map<ProjectModel, Map<String, Integer>>
-                       maximumsPerTag       // Map<String, Integer>
-                    }
-                <#assign stats = getTechReportPunchCardStats() />
-                -->
+
```
Comment (jonathanvila) [inline reporting/impl/src/main/resources/reports/templates/techReport-punchCard.ftl]: Would it make cleaner if we simply get rid of these comments ?

**Comment by m-brophy** [comment]
Ready to review again @mrizzi

**Comment by m-brophy** [comment]
@mrizzi your suggested changes now checked in

File: reporting/impl/src/main/resources/reports/resources/css/tech-report-punchcard.css [hunk 2]
```diff
@@ -166,6 +169,31 @@ table.technologiesPunchCard tbody tr {
...
+    font-style: italic;
+}
+
+.table-tooltip:hover .table-tooltiptext .tooltiptext-tech-count-header {
+    text-align: left;
+    padding-left: 0px !important;
+    font-style: italic;
 }
 
 .table-tooltip .table-tooltiptext::after {
```
[the remark this answers is not in the document]
Comment (m-brophy) [reply reporting/impl/src/main/resources/reports/resources/css/tech-report-punchcard.css]: I've realigned with bubble - but happy to remove if you still think that's the best way to go

**Pull request merged**